<launch>
  <arg name = "file_name"/>
  <node pkg="rosbag" type="play" name="rosbag" required="true" args="/media/ratchet/TOSHIBA\ EXT/kinect_bagfiles/$(arg file_name)"/>
  <node name="extract" pkg="image_view" type="extract_images" respawn="false" required="true" output="screen" cwd="ROS_HOME">
    <remap from="image" to="/kinect_2/hd/image_color_rect"/>
  </node>
</launch>
